components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityMgtConstants.java

```diff
@@ -23,6 +23,10 @@ public class IdentityMgtConstants {
 
     public static final String USER_IDENTITY_CLAIMS = "UserIdentityClaims";
     public static final String ERROR_CODE_DELIMITER = "-";
+    public static final int DEFAULT_SMS_OTP_CODE_LENGTH = 6;
```

Comment:
Is there a upper limit for the OTP length. Preferable I think 4-10 is good

Reply:
It was agreed to not have an upper limit for the OTP length since it's something that is decided by the management console user.

Reply:
Shall we not define an upper limit from the service layer? Instead, we need to make sure to restrict it from the API layer. WDYT?